### DISCUSSION TITLE
Feature or workaround to exit clink

### DISCUSSION THREAD

**Discussion by bw1faeh0** [chrisant996/clink#301] [Q&A] [answered]
Hi,

I currently have the use case, that I would run a terminal session without clink. Clink is currently configured to autorun.

I wonder if there is an other way than disable autorun in my config and restart my terminal session just to test some commands.

BR

**Comment by pukkandan** [comment, marked as answer]
`cmd /d`

**Comment by chrisant996** [comment]
Or another option is to set the `%CLINK_NOAUTORUN%` environment variable (to any non-empty value).  But that's probably harder to do programmatically on a case by case basis, versus adding the `/d` flag.

The difference is that `/d` disables _all_ autorun for the CMD instance, and `%CLINK_NOAUTORUN%` disables only autorun of Clink.
